configuration/src/gas/GAS-HARDCODES.md

```diff
@@ -0,0 +1,74 @@
+# Agent Gas Limit Values
```

Comment:
Ported from https://docs.nomad.xyz/dev/agents/gas-values.html

Reply:
this should not be in `src/`